testing/test_stepwise.py

```diff
@@ -167,11 +167,13 @@ def test_stop_on_collection_errors(broken_testdir, broken_first):
     result.stdout.fnmatch_lines("*error during collection*")
 
 
-def test_xfail_handling(testdir):
+def test_xfail_handling(testdir, monkeypatch):
     """Ensure normal xfail is ignored, and strict xfail interrupts the session in sw mode
 
     (#5547)
     """
+    monkeypatch.setattr("sys.dont_write_bytecode", True)
```

Comment:
does this still work if one forces subprocess execution? if not you may need to add the env var monkeypatch as well

Reply:
Likely not (as with other places we're doing it).
Might be worth having a method in monkeypatch for handling this in a canonical way, but I think we do not support using `--runpytest=subprocess` in our test suite currently.

Reply:
good point, lets make a follow-up issue to later decide the  future of that